### PR TITLE
Change Chinese comment to English comment

### PR DESCRIPTION
**Pull request by throcean**
We should add English comments so that we can get more developers to participate in our projects. I have added some comments and need some project members to review.

### CODE REVIEW

File: examples/chat.cc [hunk 1]
```diff
@@ -6,7 +6,7 @@ using namespace handy;
 
 int main(int argc, const char* argv[]) {
     setloglevel("TRACE");
-    map<intptr_t, TcpConnPtr> users; //生命周期比连接更长，必须放在Base前
+    map<intptr_t, TcpConnPtr> users; //the life cycle is longer than the connection and must be placed before the base
```
Comment (jovan-wong) [inline examples/chat.cc]: make comment line as an independent line

Comment (jovan-wong) [reply examples/chat.cc]: //The life cycle of `users` is longer than the connection and must be declared before `base`.

File: examples/chat.cc [hunk 2]
```diff
@@ -27,24 +27,24 @@ int main(int argc, const char* argv[]) {
             }
         });
         con->onMsg(new LineCodec, [&](const TcpConnPtr& con, Slice msg){
-            if (msg.size() == 0) { //忽略空消息
+            if (msg.size() == 0) { //ignore empty messages
```
Comment (jovan-wong) [inline examples/chat.cc]: `msg.empty()`

File: handy/codec.h [hunk 3]
```diff
@@ -4,22 +4,22 @@
 namespace handy {
 
 struct CodecBase {
-    // > 0 解析出完整消息，消息放在msg中，返回已扫描的字节数
-    // == 0 解析部分消息
-    // < 0 解析错误
+    // > 0 prase the complete message, put the message in msg, return the number of bytes scanned
```
Comment (jovan-wong) [inline handy/codec.h]: If xxx > 0, then xxx

**Comment by throcean** [comment]
I have fixed some comments.Please review it again.Thanks.

**Pull request closed without merging**